src/polyglot-notebooks-vscode-insiders/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es6",
+    "target": "ES2018",
```

Comment:
This was necessary for the named regex capture group. This version of ES is long supported in node and vscode.